tgui/packages/tgui/interfaces/NtosStationAlertConsole.js

```diff
@@ -0,0 +1,12 @@
+import { NtosWindow } from '../layouts';
+import { StationAlertConsoleContent } from './StationAlertConsole';
+
+export const NtosStationAlertConsole = () => {
+  return (
+    <NtosWindow resizable>
+      <NtosWindow.Content scrollable>
+        <StationAlertConsoleContent />
+      </NtosWindow.Content>
+    </NtosWindow>
+  );
+};
```

Comment:
Requesting you dont add this one in, in this PR, so it can be done in the full modular computers TGUI PR

Reply:
EDIT: Its already in so not an issue